### PR TITLE
refactor: add correct urls to news feed

### PR DESCRIPTION
**Pull request by britannio**
**Please follow this checklist. Please check each appropriate box (put an 'x' or check it after creating the PR).**
- [x] REQUIRED: Do you have an Issue **assigned to you** within the v1 milestone for this PR?  Put the Issue number here: https://github.com/WorldHealthOrganization/app/issues/586 and https://github.com/WorldHealthOrganization/app/issues/522
- [x] Provided detailed pull request description and a succinct title (consider template below for guidance).
- [x] Tested your changes, especially after any code review iterations.
- [x] Included any relevant screenshots of UI updates.
- [x] Followed the [Contributor Guidelines](https://github.com/WorldHealthOrganization/app/blob/master/docs/CONTRIBUTING.md).
- [x] Verified all contributions are properly licensed pursuant to the [LICENSE](https://github.com/WorldHealthOrganization/app/blob/master/LICENSE) file in the root of the repository.
- [x] Verified your name is in the [content/credits.yaml](https://github.com/WorldHealthOrganization/app/blob/master/content/credits.yaml) file (if you want it to be).

## What does this PR accomplish?
- Adds the correct urls to the news feed, sourced from the WhatsApp chatbot
- Swaps around the Padding and GestureDetector widgets in the NewsFeedItem widget to make tapping an item easier.
- Adds my name to credits.yaml

## Did you add any dependencies?
no

## How did you test the change?
Ran the app on my Android device and opened each link

### CODE REVIEW

File: client/flutter/lib/pages/news_feed.dart [hunk 1]
```diff
@@ -18,28 +16,32 @@ class NewsFeed extends StatelessWidget {
                 "Situation reports provide the latest updates on the novel coronavirus outbreak.",
             imageProvider:
                 AssetImage("assets/news_press/news_situation_reports.png"),
-            url: "https://who.int",
+            url:
```
Comment (theswerd) [inline client/flutter/lib/pages/news_feed.dart]: Where are these URLs from? do we have a spreadsheet with them yet?

Comment (britannio) [reply client/flutter/lib/pages/news_feed.dart]: The accompanying WhatsApp WHO chatbot. Check v1 Explorations in Figma to see the chatbot content and the app UI side by side.

**Comment by britannio** [comment]
URL's need to be added for every language.

File: client/flutter/lib/pages/news_feed.dart (hunk 1 above)
Comment (britannio) [reply client/flutter/lib/pages/news_feed.dart]: they're also listed here https://github.com/WorldHealthOrganization/app/issues/586

**Comment by britannio** [comment]
> Looks good!
> 
> Please get it through the CI and then I'll merge

Done

**Pull request merged**